contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -8340,6 +8340,12 @@ rewrite_string_agg_query(TSqlParser::STRING_AGGContext *ctx)
 		rewritten_query_fragment.emplace(std::make_pair(ctx->order_by_clause()->start->getStartIndex(), std::make_pair(::getFullText(ctx->order_by_clause()), "")));
 		rewritten_query_fragment.emplace(std::make_pair(ctx->RR_BRACKET()[1]->getSymbol()->getStartIndex(), std::make_pair(::getFullText(ctx->RR_BRACKET()[1]), "")));
 	}
+	
+	if (ctx->STRING_AGG())
+	{
+		size_t startPosition = ctx->STRING_AGG()->getSymbol()->getStartIndex();
+		rewritten_query_fragment.emplace(std::make_pair(startPosition, std::make_pair("", "sys.")));
+	}
```

Comment:
Please add a comment specifying the need for this change

Reply:
Updated the comment above this function.